src/SFA.DAS.Authorization.WebApi/ServicesContainerExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿#if NET462
+using System.Web.Http.Controllers;
+using System.Web.Http.ModelBinding;
+
+namespace SFA.DAS.Authorization.WebApi
+{
+    public static class ServicesContainerExtensions
+    {
+        public static void UseAuthorizationModelBinder(this ServicesContainer services)
+        {
+            services.Insert(typeof(ModelBinderProvider), 0, new AuthorizationModelBinderProvider());
```

Comment:
Do we want to introduce an interface for `AuthorizationModelBinderProvider` to implement and resolve from `DependencyResolver.Current`, like we do elsewhere in other extension classes?

Reply:
Hmm what do you think the benefit would be?